proto/perp/v1/event.proto

```diff
@@ -19,59 +19,74 @@ message PositionChangedEvent {
     // owner of the position.
     string trader_address = 2;
   
-    // amount of margin backing the position.
-    cosmos.base.v1beta1.Coin margin = 3[
+    // Amount of collateral (quote units) backing the position after the change.
+    cosmos.base.v1beta1.Coin margin = 3 [
       (gogoproto.moretags) = "yaml:\"margin\"",
       (gogoproto.nullable) = false
     ];
   
-    // margin * leverage * vPrice. 'notional' is the virtual size times the virtual price on 'vpool'.
+    // Position notional (quote units) after the change. In general, 
+    // 'notional = baseAmount * priceQuotePerBase', where size is the baseAmount.
     string position_notional = 4 [
       (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
       (gogoproto.nullable) = false];
   
-    // magnitude of the change to vsize. The vsize is the amount of base assets for the position, margin * leverage * priceBasePerQuote.
-    string exchanged_position_size = 5 [
+    // Exchanged size is the magnitude of the change to position size (base units). 
+    // The size is a signed quantity expressing how much exposure a position has in 
+    // base units of the pair. 
+    string exchanged_size = 5 [
       (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
-      (gogoproto.nullable) = false];
+      (gogoproto.nullable) = false
+    ];
+
+    /**
+     * Exchanged notional is the value of the exchanged size in quote units.
```

Comment:
Isn't the `exchangedNotional` the difference between the `positionNotional` before the change and the new `positionNotional`?